lib/generic/index.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * Express module for inbound request validations
+ */
+import fs from 'fs/promises';
+
+/**
+ * Asynchronous sleep function
+ *
+ * @param {number} millis - time to sleep (ms)
+ * @returns {Promise<void>} - Promise that resolves after millis
+ */
+export async function sleep(millis: number): Promise<void> {
```

Comment:
Update:

This shouldn't be an async function.

Reply:
what do you mean? It's meant to be used like:
```
loop:
   someFunctionCalls();
   await sleep(n);
```
The caller uses it as asyc

Reply:
The actual function (sleep) is not async, but it's returned value is.

Right now it's a function that returns a promise containing a promise (`Promise<Promise<void>>` vs `Promise<void>`)